src/1500/1549.poti

```diff
@@ -0,0 +1,10 @@
+x = leia_inteiro
+para i de 1 até x faça
+    a = leia_inteiros(2)
+    b = leia_inteiros(3)
+    media = a[2] / a[1]
+    temp1 = media * 3 * (b[2] - b[1]) / (PI * b[3]) + b[1] * b[1] * b[1]
+    temp2 = scala.math.pow(temp1, 1/3)
```

Comment:
`temp2 = temp1 ^ (1/3)`

Reply:
Arrumei aqui, não tinha encontrado na documentação que **^** fazia exponenciação. Alias é possível fazer os exercícios que usam EOF para finalizar um caso de teste com Potigol?

Reply:
É sim. A variavel `eof` indica EOF. Ela se torna `verdadeiro` quando o programa tentar ler um valor após o final da entrada. Na implementação atual há um bug que o compilador não reconhece a variável `eof` mas funciona corretamente quando o código é convertido para Scala. Para contornar, use `eof = false` quando estiver testando. Quando for enviar, remova a linha.

```
# Problema 1197
var s := leia_inteiros(" ")
# eof = false
enquanto nao eof faça
  v, t = s[1], s[2]
  escreva v * t * 2
  s := leia_inteiros(" ")
fim
```

Outra opção é ler um valor inteiro, real ou texto. Depois do EOF leia_inteiro retorna 0, leia_real retorna 0.0 e leia_texto retorna `nulo` (https://github.com/potigol/uoj-potigol/blob/master/src/1000/1026.poti).

Reply:
Obrigado, eu acabei de atualizar a documentacao para incluir a exponenciacao.